string error
int32 battery
int32 speed
int32 acceleration
int32 angular_speed


